src/components/Authentication/Signup/Signup.jsx

```diff
@@ -0,0 +1,141 @@
+    onSubmit: () => {
+      if (
+        !formik.errors.name &&
+        !formik.errors.email &&
+        !formik.errors.password
+      ) {
+        navigate("/login");
+      }
+    },
+  });
+
+  return (
+    <div className="signup flex h-screen font-sans">
+      <div className="w-3/5 bg-white text-[#1E0E62] p-4">
+        <p className="text-7xl text-left mt-[99px] ml-[115px] font-bold">
+          Sign Up to Discover Startup Features
+        </p>
+        <div className="w-[470px] h-[324px] mt-[51px] ml-[115px]">
+          <div className="name text-lg text-left mt-6">
+            <label
+              className={`block font-medium ${
+                formik.errors.name && formik.touched.name ? "text-red-500" : ""
+              }`}
+            >
+              NAME
+            </label>
+            <input
+              type="text"
+              id="name"
+              placeholder="Your name"
+              className={`border border-gray-300 w-[470px] h-[60px] rounded-full p-2 mt-[15px] pl-6 ${
+                formik.errors.name && formik.touched.name ? "text-red-500" : ""
+              }`}
+              {...formik.getFieldProps("name")}
+            />
+            {formik.errors.name && formik.touched.name ? (
+              <div className="text-red-500">{formik.errors.name}</div>
+            ) : null}
+          </div>
+          <div className="email text-lg text-left mt-6">
+            <label
+              className={`block font-medium ${
+                formik.errors.email && formik.touched.email
+                  ? "text-red-500"
+                  : ""
+              }`}
+            >
+              Email:
+            </label>
+            <input
+              type="text"
+              id="email"
+              placeholder="Your email"
+              className={`border border-gray-300 w-[470px] h-[60px] rounded-full p-2 mt-[15px] pl-6 ${
+                formik.errors.email && formik.touched.email
+                  ? "text-red-500"
+                  : ""
+              }`}
+              {...formik.getFieldProps("email")}
+            />
+            {formik.errors.email && formik.touched.email ? (
+              <div className="text-red-500">{formik.errors.email}</div>
+            ) : null}
+          </div>
+          <div className="password text-lg text-left mt-6">
+            <label
+              className={`block font-medium ${
+                formik.errors.password && formik.touched.password
+                  ? "text-red-500"
+                  : ""
+              }`}
+            >
+              Password:
+            </label>
+            <input
+              type="password"
+              id="password"
+              placeholder="Your password"
+              className={`border border-gray-300 w-[470px] h-[60px] rounded-full p-2 mt-[15px] pl-6 ${
+                formik.errors.password && formik.touched.password
+                  ? "text-red-500"
+                  : ""
+              }`}
```

Comment:
we can use [the clsx package](https://www.npmjs.com/package/clsx) to help concatenate classes in a more beautiful way